modules/apps/commerce/commerce-frontend-js/src/main/java/com/liferay/commerce/frontend/js/internal/servlet/taglib/CommerceFrontendJsDynamicInclude.java

```diff
@@ -46,7 +47,7 @@ public void include(
 
 		printWriter.print(
 			StringBundler.concat(
-				"<link href=\"", httpServletRequest.getContextPath(),
+				"<link href=\"", PortalUtil.getPathProxy(), httpServletRequest.getContextPath(),
```

Comment:
@sontruongces 
This is a Component so it'd better to make a @Reference Portal and use it